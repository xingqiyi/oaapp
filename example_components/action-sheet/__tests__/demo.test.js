import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('action-sheet');
