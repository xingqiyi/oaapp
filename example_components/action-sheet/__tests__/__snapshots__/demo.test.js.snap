// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/action-sheet/demo/basic.md correctly 1`] = `
<div
  class="actionSheetContainer"
>
  <div
    style="margin:0.15rem 0;"
  >
    <a
      aria-disabled="false"
      class="am-button"
      role="button"
    >
      <span>
        默认状态
      </span>
    </a>
  </div>
  <div
    style="margin:0.15rem 0;"
  >
    <a
      aria-disabled="false"
      class="am-button"
      role="button"
    >
      <span>
        分享功能
      </span>
    </a>
  </div>
  <div
    style="margin:0.15rem 0;"
  >
    <a
      aria-disabled="false"
      class="am-button"
      role="button"
    >
      <span>
        带多行按钮的分享功能
      </span>
    </a>
  </div>
</div>
`;
